ERCS/erc-7550.md

```diff
@@ -0,0 +1,78 @@
+---
+eip: 7550
+title: Transient ERC-20 Approvals
+description: Transient ERC-20 approvals via new contract methods
+author: Alex Kroeger (@akroeger-circle), Joey Santoro (@joeysantoro), Moody Salem (@moodysalem)
+discussions-to: TODO
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-10-26
+requires: 20, 1153
+---
+
+## Abstract
+This standard specifies a single-transaction [ERC-20](./erc-20.md) token approval mechanism as a cheaper and more secure augmentation of standard token approval functionality.
+
+It includes a separate transient (single transaction) mapping of `transientAllowance` which can be set via a call to `transientApprove`.
```

Comment:
Is `mapping` technically part of the standard, or is that an implementation detail?